src/cpp/src/image_generation/diffusion_pipeline.hpp

```diff
@@ -92,6 +92,13 @@ class DiffusionPipeline {
 
     virtual ov::Tensor decode(const ov::Tensor latent) = 0;
 
+    virtual ImageGenerationPerfMetrics get_performance_metrics() = 0;
+
+    void save_load_time(std::chrono::steady_clock::time_point start_time) {
+        auto stop_time = std::chrono::steady_clock::now();
+        m_load_time_ms = std::chrono::duration_cast<std::chrono::milliseconds>(stop_time - start_time).count();
```

Comment:
should we store it directly in `m_perf_metrics.load_time` instead of `m_load_time_ms` ?

It this case `clean_up` should not clean load time